bin/rcrestore

```diff
@@ -0,0 +1,58 @@
+#! /bin/bash
+
+if [[ ! -f $1 ]]
+then
+    echo "Usage: snap run rocketchat-server.rcrestore $SNAP_USER_COMMON/backup_file.tgz"
+    exit
+fi
+
+cd ${1%/*}
+if [[ -z $(pwd | grep "$SNAP_USER_COMMON") ]]
```

Comment:
@elespike    in my test - this won't execute in the snap; failing with permission. Pease see how to set the execute bit on this file  https://github.com/RocketChat/rocketchat-server-snap/commit/bb7b7e41c5939416a3821b8dcaf08d3c2bf1ddc7